Python/pywarpx/picmi.py

```diff
@@ -341,14 +342,16 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
         species.zmin = self.lower_bound[2]
         species.zmax = self.upper_bound[2]
 
-        # --- Only constant density is supported at this time
-        species.profile = "constant"
-        species.density = self.density
-        if density_scale is not None:
-            species.density *= density_scale
+        if self.fill_in:
+            species.do_continuous_injection = 1
 
         # --- Note that WarpX takes gamma*beta as input
-        if np.any(np.not_equal(self.rms_velocity, 0.)):
+        if (hasattr(self, "momentum_expressions")
```

Comment:
This all looks ok. However, I might imagine that a user would look at this code and decide to hack something like this
```
dist = picmi.UniformDistribution(...)
dist.momentum_expressions = ['a', 'b', 'c']
```

Reply:
We could add the `momentum_expressions` argument to `PICMI_UniformDistribution` to properly allow it. Would you like me to do that?